spec/test-utils/webrtc.ts

```diff
@@ -421,15 +422,16 @@ export function installWebRTCMocks() {
     };
 }
 
-export function makeMockGroupCallStateEvent(roomId: string, groupCallId: string): MatrixEvent {
+export function makeMockGroupCallStateEvent(roomId: string, groupCallId: string, content: IContent = {
+    "m.type": GroupCallType.Video,
+    "m.intent": GroupCallIntent.Prompt,
+    "m.terminated": false,
```

Comment:
According to MSC3401, the presence of the `m.terminated` field is supposed to be enough to mark the call as terminated, which I presume isn't what you were going for here. If the group call handler doesn't treat this as terminated, I think we should change it to do that

Reply:
Argh, good point! (typing...)